source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -231,6 +233,63 @@ def image(
     return images.clone()
 
 
+def image_features(
```

Comment:
I think this should be a manager term class that loads the model on construction and at every call you just propogate the network. Right now it seems to "hack this" into a function but for more complex cases (when you have history), you may want to have a `reset` as well for the hidden states.

Reply:
Converted to class as requested